src/images/badges/not-a-bug-a-feature.svg

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please set the font color to white, and the background color to #ef4041 to work more consistently with other badges.

In addition make sure you use the correct typography as outlined in the readme for the `A FEATURE` portion of the badge.

![image](https://user-images.githubusercontent.com/8878152/92507077-6e0e1980-f1d4-11ea-8aaa-5796807d78bf.png)